packages/terra-menu/translations/es.json

```diff
@@ -5,5 +5,6 @@
   "Terra.menu.selected": ",seleccionado",
   "Terra.menu.unselected": ",no seleccionado",
   "Terra.menu.itemWithSubmenu": ",con elementos del submenú",
-  "Terra.menu.index": "({index} de {totalItems})"
+  "Terra.menu.index": "({index} de {totalItems})",
+  "Terra.menu.navigateMenuItem": ". Para desplazarse por los elementos, presione las flechas arriba y abajo"
```

Comment:
Translations files should avoid empty spaces and characters like `.` or `,` instead you should add them on jsx file.
```jsx
let navigationContent = `, ${intl.formatMessage({ id: 'Terra.menu.navigateMenuItem' })}`;
```

Reply:
Removed `.` character from translation files and added `.` character in jsx file - [3f88409](https://github.com/cerner/terra-framework/pull/1718/commits/3f884096957816abfff6d5bd308a580b440f6582)